landing-page/src/landing-page-base-element.ts

```diff
@@ -0,0 +1,74 @@
+import type { PropertyValues } from "lit";
+import { LitElement } from "lit";
+import { property, state } from "lit/decorators";
+import {
+  computeLocalize,
+  type LandingPageKeys,
+  type LocalizeFunc,
+} from "../../src/common/translations/localize";
+import { ProvideHassLitMixin } from "../../src/mixins/provide-hass-lit-mixin";
+import type { Constructor, Resources } from "../../src/types";
+import {
+  getLocalLanguage,
+  getTranslation,
+} from "../../src/util/common-translation";
+import { computeDirectionStyles } from "../../src/common/util/compute_rtl";
+import themesMixin from "../../src/state/themes-mixin";
+import { translationMetadata } from "../../src/resources/translations-metadata";
+import type { HassBaseEl } from "../../src/state/hass-base-mixin";
+
+export class LandingPageBaseElement extends themesMixin(
```

Comment:
Why do we need themes mixin in landing page?

Reply:
To have HA styles correctly, is there a better way?
Looks like this without: 
![image](https://github.com/user-attachments/assets/a13eaa38-4615-48d3-ac2e-4e9ab367fc37)